be/src/runtime/memory/mem_tracker_limiter.h

```diff
@@ -72,7 +72,7 @@ class MemTrackerLimiter final : public MemTracker {
         if (PerfCounters::get_vm_rss() + bytes >= MemInfo::mem_limit()) {
             return Status::MemoryLimitExceeded(
                     "{}: TryConsume failed, bytes={} process whole consumption={}  mem limit={}",
-                    _label, bytes, MemInfo::current_mem(), MemInfo::mem_limit());
```

Comment:
Hi, currently memtracker only count virtual memory. And sometimes virtual memory is 3 times larger than physical memory.
I think we should include both virtual memory and physical memory in this error.

Reply:
Thank you for your prompt reply.
What I understand is to print physical memory and virtual memory separately.

Reply:
It was a bug to only print virtual memory before, because the use of physical memory to judge exceed is here.

Now be process virtual memory can be much larger than physical memory, I am investigating this.